skuba-update/skuba_update/skuba_update.py

```diff
@@ -244,7 +243,7 @@ def restart_services():
     for service in result.output.splitlines():
         cmd = run_command(['systemctl', 'restart', service], needsOutput=False)
         if cmd.returncode != 0:
-            log('Warning! Service \'{0}\' restart returned non zero exit code'.format(service))
+            log(f"Warning! Service \'{service}\' restart returned non zero exit code")
```

Comment:
The inner single quotes don't need escaped here, thanks to switching to the outer double-quotes.  Python, unlike a lot of scripting languages, doesn't handle single or double quotes differently. :)

```python
>>> service='pie'
>>> f"I like \'{service}\' with ice cream"
"I like 'pie' with ice cream"
>>> f"I like '{service}' with ice cream"
"I like 'pie' with ice cream"
```

Reply:
Ops... I forgot that. Sorry.